gulpfile.js

```diff
@@ -1,101 +1,112 @@
+gulp.task("checkDependencies", function() {
   var dependencies = buildDependenciesArray(packageJSON.dependencies);
   dependencies = dependencies.concat(
     buildDependenciesArray(packageJSON.devDependencies)
   );
 
-  checker.init({
-    start: './'
-  }, function (json, error) {
-    if (error) {
-      console.log(error);
-      process.exit(1);
-    }
-
-    var count = 0;
-    var found = [];
-    Object.keys(json).forEach(function (dependency) {
-      if (dependencies.indexOf(dependency) === -1) {
-        return;
+  checker.init(
+    {
+      start: "./"
+    },
+    function(json, error) {
+      if (error) {
+        console.log(error);
+        process.exit(1);
       }
 
-      count++;
-      found.push(dependency);
+      var count = 0;
+      var found = [];
+      Object.keys(json).forEach(function(dependency) {
+        if (dependencies.indexOf(dependency) === -1) {
+          return;
+        }
 
-      var licenses = json[dependency].licenses;
-      if (licenses === 'UNKNOWN') {
-        console.warn(
-          'Dependency has an unknown license', json[dependency]
-        );
-      }
+        count++;
+        found.push(dependency);
 
-      if (licenses.indexOf('LGPL') === -1 && licenses.match(/GPL/gi)) {
-        console.log('Package has invalid license.', json[dependency]);
-        process.exit(1);
-      }
-    });
+        var licenses = json[dependency].licenses;
+        if (licenses === "UNKNOWN") {
+          console.warn("Dependency has an unknown license", json[dependency]);
+        }
 
-    // Ensure we found them all
-    if (count !== dependencies.length) {
-      console.log('Dependency length doesn\'t match.');
-      var missing = dependencies.filter(function (dependency) {
-        return found.indexOf(dependency) === -1;
+        if (licenses.indexOf("LGPL") === -1 && licenses.match(/GPL/gi)) {
+          console.log("Package has invalid license.", json[dependency]);
+          process.exit(1);
+        }
       });
-      console.log(missing);
-      process.exit(1);
+
+      // Ensure we found them all
+      if (count !== dependencies.length) {
+        console.log("Dependency length doesn't match.");
+        var missing = dependencies.filter(function(dependency) {
+          return found.indexOf(dependency) === -1;
+        });
+        console.log(missing);
+        process.exit(1);
+      }
     }
-  });
+  );
 });
 
-gulp.task('serve', function () {
+gulp.task("serve", function() {
   connect.server({
     port: 4200,
-    root: './dist'
+    root: "./dist"
   });
 });
 
 // remove 'fsevents' from shrinkwrap, since it causes errors on non-Mac hosts
 // see https://github.com/npm/npm/issues/2679
-gulp.task('fixShrinkwrap', function (done) {
+gulp.task("fixShrinkwrap", function(done) {
   delete shrinkwrap.dependencies.fsevents;
-  var shrinkwrapString = JSON.stringify(shrinkwrap, null, '  ') + '\n';
-  fs.writeFile('./npm-shrinkwrap.json', shrinkwrapString, done);
+  delete shrinkwrap
```

Comment:
I believe this is the only meaningful line in this PR! The rest is string formatting 😄